styleguide/package.json

```diff
@@ -1,45 +1,44 @@
 {
   "name": "ama-style-guide",
-  "version": "1.13.5",
+  "version": "3.5.0",
```

Comment:
This is tagging the release. What was the reasoning from going from `1.13.5` to `3.5.0`?